docs/source/Support.md

```diff
@@ -20,17 +20,17 @@ NOTE: this table is a start, it needs to be extended.
 
 Feature                        | C++    | Java   | C#     | Go     | Python | JS        | C    | PHP | Ruby
 ------------------------------ | ------ | ------ | ------ | ------ | ------ | --------- | ---- | --- | ----
-Codegen for all basic features | Yes    | Yes    | Yes    | Yes    | Yes    | Yes       | WiP  | WiP | WiP
-JSON parsing                   | Yes    | No     | No     | No     | No     | No        | No   | No  | No
+Codegen for all basic features | Yes    | Yes    | Yes    | Yes    | Yes    | Yes       | Yes  | WiP | WiP
+JSON parsing                   | Yes    | No     | No     | No     | No     | No        | Yes  | No  | No
 Simple mutation                | Yes    | WIP    | WIP    | No     | No     | No        | No   | No  | No
-Reflection                     | Yes    | No     | No     | No     | No     | No        | No   | No  | No
-Buffer verifier                | Yes    | No     | No     | No     | No     | No        | No   | No  | No
-Testing: basic                 | Yes    | Yes    | Yes    | Yes    | Yes    | Yes       | ?    | ?   | ?
-Testing: fuzz                  | Yes    | No     | No     | Yes    | Yes    | No        | ?    | ?   | ?
+Reflection                     | Yes    | No     | No     | No     | No     | No        | Basic| No  | No
+Buffer verifier                | Yes    | No     | No     | No     | No     | No        | Yes  | No  | No
+Testing: basic                 | Yes    | Yes    | Yes    | Yes    | Yes    | Yes       | Yes  | ?   | ?
+Testing: fuzz                  | Yes    | No     | No     | Yes    | Yes    | No        | No   | ?   | ?
 Performance:                   | Superb | Great  | Great  | Great  | Ok     | ?         |Superb| ?   | ?
-Platform: Windows              | VS2010 | Yes    | Yes    | ?      | ?      | ?         | ?    | ?   | ?
-Platform: Linux                | GCC282 | Yes    | ?      | Yes    | Yes    | ?         | ?    | ?   | ?
-Platform: OS X                 | Xcode4 | ?      | ?      | ?      | Yes    | ?         | ?    | ?   | ?
+Platform: Windows              | VS2010 | Yes    | Yes    | ?      | ?      | ?         | No   | ?   | ?
```

Comment:
FlatCC also tries really hard, hence the include/flatcc/portable library, but everything can't be covered at once by one developer. I'll look onto getting a Windows VM running. It would be great if Google could somehow provide a big-endian testing platform.